### ISSUE TITLE
Missing XLS file for ms09_067_excel_featheader

### ISSUE THREAD

**Issue by wchen-r7** [rapid7/metasploit-framework#4695]
Really easy to reproduce:

```
msf exploit(ms09_067_excel_featheader) > run

[*] Creating Excel spreadsheet ...
[-] Exploit failed: Errno::ENOENT No such file or directory @ rb_sysopen - /Users/sinn3r/.msf4/data/exploits/msf.xls
````

And you get:

```
[02/02/2015 01:23:07] [d(1)] core: Module x86/shikata_ga_nai is compatible with windows/fileformat/ms09_067_excel_featheader
[02/02/2015 01:23:07] [i(0)] core: windows/meterpreter/reverse_tcp: iteration 1: Successfully encoded with encoder x86/shikata_ga_nai (size is 308)
[02/02/2015 01:23:07] [e(0)] core: Exploit failed (windows/fileformat/ms09_067_excel_featheader): Errno::ENOENT No such file or directory @ rb_sysopen - /Users/sinn3r/.msf4/data/exploits/msf.xls
[02/02/2015 01:23:07] [d(3)] core: Call stack:
/Users/sinn3r/rapid7/msf/lib/rex/ole/storage.rb:57:in `initialize'
/Users/sinn3r/rapid7/msf/lib/rex/ole/storage.rb:57:in `new'
/Users/sinn3r/rapid7/msf/lib/rex/ole/storage.rb:57:in `open'
/Users/sinn3r/rapid7/msf/lib/rex/ole/storage.rb:30:in `initialize'
/Users/sinn3r/rapid7/msf/modules/exploits/windows/fileformat/ms09_067_excel_featheader.rb:150:in `new'
/Users/sinn3r/rapid7/msf/modules/exploits/windows/fileformat/ms09_067_excel_featheader.rb:150:in `exploit'
/Users/sinn3r/rapid7/msf/lib/msf/core/exploit_driver.rb:205:in `job_run_proc'
/Users/sinn3r/rapid7/msf/lib/msf/core/exploit_driver.rb:166:in `run'
/Users/sinn3r/rapid7/msf/lib/msf/base/simple/exploit.rb:136:in `exploit_simple'
/Users/sinn3r/rapid7/msf/lib/msf/base/simple/exploit.rb:161:in `exploit_simple'
/Users/sinn3r/rapid7/msf/lib/msf/ui/console/command_dispatcher/exploit.rb:111:in `cmd_exploit'
/Users/sinn3r/rapid7/msf/lib/rex/ui/text/dispatcher_shell.rb:427:in `run_command'
/Users/sinn3r/rapid7/msf/lib/rex/ui/text/dispatcher_shell.rb:389:in `block in run_single'
/Users/sinn3r/rapid7/msf/lib/rex/ui/text/dispatcher_shell.rb:383:in `each'
/Users/sinn3r/rapid7/msf/lib/rex/ui/text/dispatcher_shell.rb:383:in `run_single'
/Users/sinn3r/rapid7/msf/lib/rex/ui/text/shell.rb:200:in `run'
/Users/sinn3r/rapid7/msf/lib/metasploit/framework/command/console.rb:38:in `start'
/Users/sinn3r/rapid7/msf/lib/metasploit/framework/command/base.rb:82:in `start'
./msfconsole:48:in `<main>'
```

There is no msf.xls in msf/data/exploits, where did it go?

**Comment by wchen-r7** [comment]
Hmmm it looks like there was never a default msf.xls. Should supply a default one.

**Comment by jvazquez-r7** [comment]
PR with patch here: https://github.com/rapid7/metasploit-framework/pull/4759

**Comment by wchen-r7** [comment]
Fixed in #4759
